docs/snippets/defaults_consumer_device.mdx

```diff
@@ -0,0 +1,26 @@
+```
+default_device_watts:
+  scope3:
```

Comment:
Probably should then not have then also duplicated in `templates/end_user_devices`

Reply:
agree - was going to zap all of the overlapping defaults but wasn't sure how we wanted to handle the migration. Thought maybe we'd flip the API over to the new ones first?

Reply:
Yeah. There are things relying on it for sure. Let's merge this and then we'll convert the pipeline and then nuke it.